forms/src/app/login-form/login-form.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.css']
+})
+export class LoginFormComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onSubmit(value) {
+    console.log(JSON.stringify(value));
```

Comment:
I thought about this, nice suggestion. Since we are already in devtools it shouldn't be too hard to find the console. Nothing else is getting put there, so don't need to worry about output getting buried. Also the focus is on Augury, not so much on functionality, the stuff that happens when the User clicks submit.